SpatialGDK/Documentation/content/porting-unreal-project-to-gdk.md

```diff
@@ -182,22 +157,17 @@ For example:
         
     
 ## Generating code for SpatialOS C++ workers and building your project
-1. In your terminal, navigate to `<ProjectRoot>\<GameRoot>\Scripts` and run `Codegen.bat`.
-This initializes the project. It should succeed quickly and silently.
 1. Set your Unreal project to work with the SpatialOS Unreal GDK fork of the Unreal Engine, which you cloned and installed earlier. To do this:
     1. In File Explorer, navigate to `<ProjectRoot>\<GameRoot>`.
     1. Right-click your uproject file and select **Switch Unreal Engine version**.
     1. Select the path to the Unreal Engine fork you cloned earlier.
-    1. In your terminal, from `<ProjectRoot>\<GameRoot>\Scripts`, run `BuildWorker.bat <YourProject>Editor Win64 Development <YourProject>.uproject`
+    1. In your terminal, from `<UnrealGDKRoot>\SpatialGDK\Build\Scripts`, run `BuildWorker.bat <YourProject>Editor Win64 Development <YourProject>.uproject`
```

Comment:
Maybe `<ProjectRoot>\<GameRoot>\UnrealGDK\Plugins\SpatialGDK\Build\Scripts`

Reply:
Changed it to `Plugins\<UnrealGDKRoot>\SpatialGDK\Build\Scripts`